nuxeo-services/nuxeo-platform-convert/src/main/java/org/nuxeo/ecm/platform/convert/ooomanager/ConfigBuilderHelper.java

```diff
@@ -54,15 +55,12 @@ protected static void hackClassLoader() throws IOException {
             System.arraycopy(paths, 0, tmp, 0, paths.length);
             tmp[paths.length] = ldPath;
             field.set(null, tmp);
-            System.setProperty("java.library.path",
-                    System.getProperty("java.library.path"));
+            System.setProperty("java.library.path", System.getProperty("java.library.path"));
```

Comment:
I don't understand the purpose of this line. Anyway, it's from 5.4, not your concern.